R/endpoints.R

```diff
@@ -701,15 +701,15 @@ covidcast <-
     if (!missing(issues) && !missing(lag)) {
       stop("`issues` and `lag` are mutually exclusive")
     }
-    check_single_string_param("data_source", data_source)
-    check_string_param("signals", signals)
-    check_single_string_param("time_type", time_type)
-    check_single_string_param("geo_type", geo_type)
-    check_epirange_param("time_values", time_values)
-    check_string_param("geo_values", geo_values)
-    check_single_epirange_param("as_of", as_of, FALSE)
-    check_epirange_param("issues", issues, FALSE)
-    check_single_int_param("lag", lag, FALSE)
+    check_character_param("data_source", data_source)
+    check_character_param("signals", signals)
+    check_character_param("time_type", time_type)
+    check_character_param("geo_type", geo_type)
```

Comment:
question: original "single string" and "string" checks are replaced by "character" check here.  Is this okay, or should there be a (rlang-definition) string/scalar_character check?

Reply:
(same question for other endpoints as well..)

Reply:
Yea, I made a mistake here, fix incoming!